### PR TITLE
Apply various changes to README.md

### PR DESCRIPTION
**Pull request by Southparkfan**
It's been a while since I wrote this draft, all changes since then should have been applied properly in this PR. Feel free to leave comments :-)

### CODE REVIEW

**Comment by jelletreep** [comment]
Thanks so much! I will go over the changes

File: README.md [hunk 1]
```diff
@@ -2,91 +2,87 @@
...
+Membership of the Utrecht University GitHub organization, including access to internal repositories, is restricted to employees of Utrecht University (UU). You can use your existing personal GitHub account; if you do not have an account yet, please create one. During this process, you will have to authenticate your personal GitHub account by using your Solis-id. Your Solis-id is merely used to verify your employee status; all commits and actions performed in GitHub will be attributed to your personal GitHub account. By using one GitHub account, your GitHub profile page (which can be used as your curriculum vitae) will not only show your contributions in repositories belonging to this GitHub organization, but contributions in different GitHub organizations or personal repositories as well.
```
Comment (jelletreep) [inline README.md:11]: Great revision! Could change the following sentence?
"You can use your existing personal GitHub account" > "Become a member using your existing personal GitHub account"

Comment (jelletreep) [inline README.md:11]: Suggestion: Would it be better to change the authentication sentence to:
"you will have to authenticate for this organization using your Solis-id"?
I think it is important for the reader to know that the solis id is only needed for the organization and will not alter any other normal interactions with e.g. personal repositories in github

**Review by jelletreep — COMMENTED** [review]
Thanks a lot for this pull request! I think it improves clarity and readability. I have suggested some minor changes

File: README.md (hunk 1 above)
Comment (Southparkfan) [reply README.md:11]: Sure, will change.

Comment (Southparkfan) [reply README.md:11]: Fair point, I'll change that.

File: README.md [hunk 2]
```diff
@@ -2,91 +2,87 @@
...
 
 ### Code and non-sensitive data
-This GitHub organization is meant for managing source code. GitHub is not suited to store datasets with personal and/or sensitive data. However, it is sometimes used for version control of documents or small sized, non-sensitive datasets. For research data, make sure that you create a data management plan first, and choose an [appropriate storage solution](https://www.uu.nl/en/research/research-data-management/tools-services/tools-for-storing-and-managing-data/storage-solutions).
+This GitHub organization is meant for managing source code. GitHub is not suited to store datasets with personal data or sensitive data exceeding the security level of this GitHub organization. However, it may be used for version control of documents or small-sized datasets. For research data, make sure that you create a data management plan first, and choose an [appropriate storage solution](https://www.uu.nl/en/research/research-data-management/tools-services/tools-for-storing-and-managing-data/storage-solutions). For other objects, you could use the [Tooladvisor](https://tools.uu.nl/tooladvisor/).
```
[the remark this answers is not in the document]
Comment (Southparkfan) [reply README.md:42]: I'll follow up with you personally (with regard to my reply).

[the remark this answers is not in the document]
Comment (Southparkfan) [reply README.md:42]: Discussed :)

**Comment by Southparkfan** [comment]
> Thank you for the changes! I committed some minor edits directly to your branch in the section 'code-and-non-sensitive-data' . Let me know if you agree and if I can merge!

Go for it :)

**Pull request merged**